### ISSUE TITLE
Add schema validation to make_executable_schema

### ISSUE THREAD

**Issue by rafalp** [mirumee/ariadne#251]
GraphQL-Core-next implements `assert_valid_schema` function in `graphql.type.validate` that takes `GraphQLSchema` and validates its correctness.

Currently, this function is called as part of query execution, during query validation, but we could also run it inside the `make_executable_schema`, so developers get this error on application's initialization.

**Comment by caladd** [comment]
I'll work on this.

**Comment by rafalp** [comment]
@caladd thank you for stepping up to contribute! I've assigned the task to you.

**Comment by caladd** [comment]
@rafalp Do you have an example of a schema error that currently isn't getting caught?

Inside of `make_executable_schema` the calls to `parse` and `build_and_extend_schema` seem to catch any invalid schema I can think of. (Granted, I'm relatively new to GraphQL).

**Comment by rafalp** [comment]
You can pass valid GraphQL string to `make_executable_schema` that's not a schema and it won't produce an error:

```python
type = """
query MyQuery { hello world }
"""

schema = make_executable_schema(type)  # will return query object
```

You can also define schema without query type, which is impossible to run queries against:

```python
type = """
type Mutation {
    login(username: String!, password: String!): Boolean
}
"""

schema = make_executable_schema(type)  # will return invalid schema
```

**Comment by caladd** [comment]
Thanks! I was passing a simple bindable, which was catching those type of errors. Without bindables I seem them slip though. I'll have a PR to fix that soon.

**Comment by rafalp** [comment]
Fixed via #254
